services/outputhost/cgcache.go

```diff
@@ -405,6 +411,11 @@ func (cgCache *consumerGroupCache) manageConsumerGroupCache() {
 			cgCache.logger.Info("stopped")
```

Comment:
Since we're doing a bunch of stuff here, probably useful to add a log line after everything is done